packages/beacon-node/src/api/impl/beacon/state/index.ts

```diff
@@ -85,6 +86,9 @@ export function getBeaconStateApi({
     },
 
     async getStateValidators({stateId, validatorIds = [], statuses = []}) {
+      ensureUniqueItemsOrThrow(validatorIds, "Duplicate validator IDs provided");
+      ensureUniqueItemsOrThrow(statuses, "Duplicate statuses provided");
```

Comment:
this only needs to be checked if `length > 0` which we do below for each, should move those there